components/camel-stomp/src/main/java/org/apache/camel/component/stomp/StompEndpoint.java

```diff
@@ -137,6 +144,10 @@ public void run() {
 
     protected void send(final Exchange exchange, final AsyncCallback callback) {
         final StompFrame frame = new StompFrame(SEND);
+        
+        // feature for CAMEL-7672
```

Comment:
We dont really need comments to CAMEL tickets people can look in git commit history